src/main/webapp/WEB-INF/jsp/question/questionEditForm.jsp

```diff
@@ -8,7 +8,7 @@
 
 <h2 class="title subheader page-title">${t['question.edit_form.title']}</h2>
 
-<form class="validated-form question-form hinted-form"
+<form class="validated-form question-form hinted-form form-with-upload"
```

Comment:
I'm guessing this class should only be here if Globals.inHouseUploading is true, am I right?